kong/plugins/oauth2/access.lua

```diff
@@ -831,6 +831,11 @@ local function retrieve_token(conf, access_token)
         [ERROR] = "invalid_token",
         error_description = "The access token is global, but the current " ..
           "plugin is configured without 'global_credentials'",
+      },
+      {
+        ["WWW-Authenticate"] = 'Bearer realm="service" error=' ..
```

Comment:
Did we decide to hardcode realm=service? Or should it be dynamic? Or is there no good way to figure it out? Which would me we’d need to add `config.realm` so that it could be specified?